rules/apps.libsonnet

```diff
@@ -100,6 +127,15 @@
               )
             ||| % $._config,
           },
+          {
+            record: 'container_cpu:kube_pod_container_resource_limits',
```

Comment:
nit: I am not exactly sure about the prefix. @metalmatze @povilasv any takes on this?

Reply:
Agreed, I think is should be something like `cluster:namespace:pod:active:kube_pod_container_resource_limits`

Reply:
:+1: I like @brancz suggestion, let's rename it.

Reply:
I don't agree, with name `cluster:namespace:pod:active:kube_pod_container_resource_limits` we are loosing information that this is about CPU and we have immediate conflict with similar rule for memory - https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/639/files#diff-bc8c24b130076f1ca75adec5baee841ea4bed3a51a5d72b4437e31dd1d188a5cR108-R115

Reply:
How about `cluster:namespace:pod_cpu:active:kube_pod_container_resource_limits `, `cluster:namespace:pod_memory:active:kube_pod_container_resource_limits ` ?

Reply:
Whops, yes my bad, cpu should definitely be in there. I'd be happy with your latest suggestion @PhilipGough!